### DISCUSSION TITLE
Details on Calculating and Maintaining the Optimum Staking Ratio

### DISCUSSION THREAD

**Discussion by boredray** [pkgxdev/pkgx#921] [Help] [answered]
In the white paper, it's mentioned that the tea Protocol aims to implement an optimum staking ratio to prevent a small number of highly-staked packages from collecting most staking rewards. Could you provide more details on how this optimum staking ratio will be calculated and adjusted? Additionally, what measures will be in place to ensure a fair distribution of staking rewards among packages with varying degrees of staked TEA tokens?

**Comment by mxcl** [comment, marked as answer]
This is the wrong github, go to https://github.com/teaxyz
